include/librealsense2/h/rs_option.h

```diff
@@ -96,6 +96,7 @@ extern "C" {
         RS2_OPTION_THERMAL_COMPENSATION, /**< Depth Thermal Compensation for selected D400 SKUs */
         RS2_OPTION_TRIGGER_CAMERA_ACCURACY_HEALTH, /**< Enable depth & color frame sync with periodic calibration for proper alignment */
         RS2_OPTION_RESET_CAMERA_ACCURACY_HEALTH,
+        RS2_OPTION_ENABLE_WEAK_USB_HOST_WA, /**< Enable workaround for weak USB hosts to improve performance and stability */
```

Comment:
Pls rename and refactor the option to be rs2_option_trb_size so that it will read/set the TRB size (units TBD) - it is not platform specfic and therefore may be utilized in additional scenarios.
You can set an enum for the supported RGB sizes in order to enforce strict (and validated) values